### PR TITLE
Update pipeline_controlnet_sd_xl.py

### PR DESCRIPTION
**Pull request by nbardy**
# What does this PR do?

Add denoising_end to controlnet SDXL

## Who can review

- Pipelines:  @patrickvonplaten and @sayakpaul

### CODE REVIEW

File: src/diffusers/pipelines/controlnet/pipeline_controlnet_sd_xl.py [hunk 1]
```diff
@@ -755,6 +755,7 @@ def __call__(
         self,
         prompt: Union[str, List[str]] = None,
         prompt_2: Optional[Union[str, List[str]]] = None,
+        denoising_end: Optional[float] = None,
```
Comment (sayakpaul) [inline src/diffusers/pipelines/controlnet/pipeline_controlnet_sd_xl.py:758]: Let's also add it in the docstrings.

**Review by sayakpaul — APPROVED** [review]
Thanks for your contributions!

Let's also add some tests to ensure the robustness of `denoising_end` in the SDXL ControlNet?

Additionally, have you played with `denoising_end` for the SDXL ControlNet pipeline? If so, would love to see some visual examples.

**Review by patrickvonplaten — COMMENTED** [review]
+1 on what Sayak said. After this we can merge I think.

**Comment by nbardy** [comment]
Going to close this. The experiments didn't pan out. No need to upstream.

**Pull request closed without merging**